Classes/Utility/SegmentControl+Border.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  SegmentControl+Border.swift
+//  Freetime
+//
+//  Created by Ehud Adler on 3/12/19.
+//  Copyright © 2019 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+extension UISegmentedControl {
+    func removeBorders() {
+        setBackgroundImage(imageWithColor(color: backgroundColor ?? .white), for: .normal, barMetrics: .default)
+        setBackgroundImage(imageWithColor(color: tintColor ?? .white), for: .selected, barMetrics: .default)
+        setDividerImage(imageWithColor(color: UIColor.clear), forLeftSegmentState: .normal, rightSegmentState: .normal, barMetrics: .default)
+    }
+
+    // create a 1x1 image with this color
+    private func imageWithColor(color: UIColor) -> UIImage {
+        let rect = CGRect(x: 0.0, y: 0.0, width: 1.0, height: 1.0)
+        UIGraphicsBeginImageContext(rect.size)
```

Comment:
You should be able to use `UIGraphicsImageRenderer` here instead, which removes the need to have to deal with optionals.